src/components/button/Button.tsx

```diff
@@ -318,6 +319,7 @@ type ButtonBase = {
   className?: string;
   color?: Color | 'inherit';
   disabled?: boolean;
+  hoverBackground?: Color;
```

Comment:
I forget why we allowed 'inherit' as an option, but it might be a good idea to add it for the hoverBackground as well? I honestly don't know, but it couldn't hurt